DiscountDiscover/Managers/APIManager.m

```diff
@@ -54,7 +54,7 @@ - (void)fetchDeals:(double)radius completion:(void(^)(NSArray<Deal *> *deals, NS
 }
 
 - (void)fetchNearbyDeal:(void (^)(Deal *deal, UIBackgroundFetchResult result))completionHandler {
-    [self fetchDeals:3 completion:^(NSArray<Deal *> *deals, NSError *error) {
+    [self fetchDeals:0.2 completion:^(NSArray<Deal *> *deals, NSError *error) {
```

Comment:
Oh, I see this is the radius and not the number of deals. You should make this explicit in the method signature though though, like `fetchDealsWithRadius:completion:`. Also you should pass a limit to the Discount API so it only fetches one deal if that's all you need, you can use the `per_page` query parameter for this, it defaults to 10 so you're always fetching 10 deals here. You can also use the `order` query param to sort by location if you want to get the closest deal.